scheduler/wmap/wmap.go

```diff
@@ -365,9 +369,10 @@ func (p *ProcessWorkflowMapNode) GetConfigNode() (*cdata.ConfigDataNode, error)
 }
 
 type PublishWorkflowMapNode struct {
-	Name    string `json:"plugin_name"yaml:"plugin_name"`
-	Version int    `json:"plugin_version"yaml:"plugin_version"`
-	// TODO publisher config
```

Comment:
I can add it back but I really don't think it's still relevant. We can specify a config within a publisher.